### DISCUSSION TITLE
MongoDB Engine, deleting many-to-many documents

### DISCUSSION THREAD

**Discussion by tal-rofe** [prisma/prisma#13652] [Q&A]
I have the following schema:
```prisma
model Group {
  id String @id @default(auto()) @map("_id") @db.ObjectId

  policyIDs      String[]       @db.ObjectId

  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt

  policies Policy[] @relation(fields: [policyIDs], references: [id])
}

model Policy {
  id String @id @default(auto()) @map("_id") @db.ObjectId

  groupsIDs     String[]        @db.ObjectId

  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt

  groups Group[] @relation(fields: [groupsIDs], references: [id])
}
```

So I have `Policy` and `Group` models. I want to delete a group, and automatically remove its Id from any `Policy` which holds the Id in its `groupsIDs` slice.
If I do: `.delete({ where: { id: groupId } }),` will it work automatically out of the box? Or should I remove the Id of the slices manually?

**Comment by austincrim** [comment]
Hey @tal-rofe 👋,

Typically, you would use referential actions for this, but I don't believe they are supported yet for this specific scenario.

Would you mind filling out a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for this use case? Thanks!

**Reply by tal-rofe** [reply]
@austincrim There is already an opened issue for this case. Is there any progress?

**Reply by austincrim** [reply]
The existing issue will have the most up-to-date info 👍. Feel free to add your +1, though.

**Reply by tal-rofe** [reply]
@austincrim 

Could you please share with me the issue's link? can't find it

**Reply by austincrim** [reply]
I can't find an existing one either, I assumed you had the link since you mentioned it was an existing issue.

Feel free to [open a new issue](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
